src/pages/cocktail/[id]/index.js

```diff
@@ -0,0 +1,142 @@
+import {
+  Container,
+  Typography,
+  Button,
+  CardMedia,
+  Card,
+  CardContent,
+  Box,
+  CircularProgress,
+} from '@material-ui/core';
+import { useRouter } from 'next/router';
+import { useEffect, useState } from 'react';
+import Header from '../../../components/Header';
+
+function Redirect({ to }) {
+  const router = useRouter();
+
+  useEffect(() => {
+    router.push(to);
+  }, [router, to]);
+
+  return null;
```

Comment:
why you need to return `null` here?

Reply:
Needs retunrn statement. No need fo rnull spe